src/resources/authoring/gv2.json

```diff
@@ -408,7 +408,7 @@
       },
       {
         "comment": "*** FV Test Level 2 ***",
-        "challengeId": "gamete-1target-simple",
+        "challengeId": "egg-breeding-basic-match-visible",
```

Comment:
`gamete-1target-simple` is the ID that Trudi came up with, while `egg-breeding-basic-match-visible` is what I was using before I had anything from her. The ITS server can handle `gamete-1target-simple`, so I think we should stick with that